ui/components/MesheryPatterns.js

```diff
@@ -182,7 +227,8 @@ function MesheryPatterns({
 
   const searchTimeout = useRef(null);
   /**
-   * fetch patterns when the page loads
+   * fetch patterns when the page loadsOh that's okay @warunicorn19
```

Comment:
seemed to have commented this by mistake😅

Reply:
How I got these changes? I am confused. Hell, When did I do this

Reply:
Oh, I think it's a surprise from ctrl+c and ctr+v

Reply:
Removed